inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -130,11 +132,56 @@ void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
     }
 }
 
+NodeVector get_ordered_ops(const ngraph::Function& f) {
```

Comment:
I have a question. As I remember from the discussions, we couldn't achieve the same order as in the MO.
In this case I have a question. Why do we need a this method and cannot use the default nGraph order?

Reply:
We talked offline. Conclusion is that because current algorithm is not fully compliant with MO we don't want to introduce it to codebase yet. Ticket will be deferred and PR closed without merging.  We can revisit this topic oneday if time allows or it proves to be more important than we think of it right now.